### ISSUE TITLE
most_common_interests_with(user_id) from Python 3 Introductions file 

### ISSUE THREAD

**Issue by snooze** [joelgrus/data-science-from-scratch#47]
I think that:

> def most_common_interests_with(user_id):
>     return Counter(interested_user_id
>         for interest in interests_by_user_id**["user_id"]**
>         for interested_user_id in user_ids_by_interest[interest]
>         if interested_user_id != user_id)

Should be:

> def most_common_interests_with(user_id):
>     return Counter(interested_user_id
>         for interest in interests_by_user_id**[user_id]**
>         for interested_user_id in user_ids_by_interest[interest]
>         if interested_user_id != user_id)

**Comment by Neshmwaniki** [comment]
Agree, the former assumes integers are subscriptable @snooze good catch!
